### DISCUSSION TITLE
Incompatibility with @wagmi/chains after npm init wagmi

### DISCUSSION THREAD

**Discussion by 0xLukin** [wevm/wagmi#1473] [Q&A] [answered]
I used npm init wagmi to create a vite-rainbowkit project. When I imported @wagmi/chains and bsc wanted to add it to configureChains, I reported an attribute "rpcUrls" incompatibility error.

It seems that the exported bsc can't be used directly, there are some format errors, but my ts foundation is very poor, hope someone can help me

`
import { bsc } from "@wagmi/chains"

const { chains, provider, webSocketProvider } = configureChains(
  [bsc],
  [publicProvider(),],
)
`

**Comment by 0xLukin** [comment, marked as answer]
I think I understand the problem, rainbowkit is still using version 0.8.x, and the latest built-in chain needs version 0.9.x.

**Reply by jxom** [reply]
Can you try update `wagmi` to `^0.9.0`, and `@rainbow-me/rainbowkit` to `^0.8.1`?

**Reply by 0xLukin** [reply]
yeh, I try. Thank you very much! Successfully solved my problem, maybe npm init wagmi needs to be updated, the template veite-rainbow uses @wagmi 0.8.x by default, and needs to be updated to 0.9.0.
